erpnext/setup/doctype/employee/employee.json

```
@@ -817,14 +819,27 @@
    "fieldname": "iban",
    "fieldtype": "Data",
    "label": "IBAN"
+  },
+  {
+   "fieldname": "country",
+   "fieldtype": "Link",
+   "label": "Country",
+   "options": "Country",
+   "reqd": 1
+  },
+  {
+   "fieldname": "state",
+   "fieldtype": "Data",
+   "label": "State",
+   "mandatory_depends_on": "eval:doc.country==\"United States\""
```

```suggestion
   "options": "Country"
  },
  {
   "fieldname": "state",
   "fieldtype": "Data",
   "label": "State"
```

Remove these reqd checks

---

https://github.com/frappe/erpnext/pull/41392 
mam, New PR by removing the reqd checks

---

Push it to this branch itself?

---

Mam, Pushed to the Same Branch.